src/events/processTransfers/index.js

```diff
@@ -29,10 +29,15 @@ function processTransfersBuilder(config) {
       validatorContract = new web3Home.eth.Contract(bridgeValidatorsABI, validatorContractAddress)
     }
 
+    rootLogger.debug(`Processing ${transfers.length} Transfer events`)
     const callbacks = transfers.map(transfer =>
       limit(async () => {
         const { from, value } = transfer.returnValues
 
+        const logger = rootLogger.child({
+          eventTransactionHash: transfer.transactionHash
+        })
+
         let gasEstimate
         try {
           gasEstimate = await estimateGas({
```

Comment:
It makes sense to add logging within `estimateGas` to return the value received by the the corresponding JSON-RPC call. It will also allow to check that it was not stuck there.
Moreover there are lot of call to RPC in `estimateGas`. So it seems reasonable to add logging after every call there.

Reply:
Added more logs in b2b75daa0c9acc27a43145461e64963ebad290ae.